examples/game-clock/src/main/kotlin/com/justai/jaicf/examples/gameclock/GameController.kt

```diff
@@ -62,7 +62,7 @@ class GameController(context: BotContext) {
 
     fun record(timeMs: Long) {
         currentTime = timeMs
-        overall = overall?.let { it.plus(timeMs) } ?: timeMs
+        overall = overall?.plus(timeMs) ?: timeMs
```

Comment:
Maybe something like that?
```suggestion
        overall = (overall ?: 0L) + timeMs
```

Or, we can even get rid of nullability using `withDefault`:
`var overall: Long by context.client.withDefault { 0L }`